protocols/luci-proto-batman-adv/htdocs/luci-static/resources/protocol/batadv.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+'require form';
+'require network';
+
+
+network.registerPatternVirtual(/^bat\d+/);
+
+return network.registerProtocol('batadv', {
+	getI18n: function() {
+		return _('B.A.T.M.A.N. advanced');
+	},
+
+	getIfname: function() {
+		return this._ubus('l3_device') || this.sid;
+	},
+
+	getOpkgPackage: function() {
+		return 'batman-adv';
+	},
+
+	isFloating: function() {
+		return true;
+	},
+
+	isVirtual: function() {
+		return true;
+	},
+
+	getDevices: function() {
+		return null;
+	},
+
+	containsDevice: function(ifname) {
+		return (network.getIfnameOf(ifname) == this.getIfname());
+	},
+
+	renderFormOptions: function(s) {
+		var dev = this.getL3Device() || this.getDevice(), o;
+
+
+		try {
+			s.tab('mesh', _('Mesh Routing'), _('Mesh and routing related options'));
+		}
+		catch(e) {}
+
+
+
+// @FIXME - the list of routing protocols should not be hard coded but come from batctl  
+
+		o = s.taboption('mesh', form.ListValue, 'routing_algo', _('Routing Algorithm'),_('The algorithm that is used to discover mesh routes'));
+		o.value('BATMAN_IV', 'BATMAN_IV');
+		o.value('BATMAN_V', 'BATMAN_V');
+		o.default = 'BATMAN_IV';
+
+		o = s.taboption('mesh', form.Flag, 'aggregated_ogms', _('aggregate originator messages'),_('reduces overhead by collecting and aggregating originator messages in a single packet rather than many small ones'));
```

Comment:
Could you please break the long lines at the last argument into a new line, in the whole file. They are quite long.
Regardless of whether you break now, a space after the comma is missing here.

Can you also capitalise the first char in the 3 argument over all in the whole function tapoption? Also, it wouldn't hurt to find a more meaningful expression.
- aggregate originator messages -> Aggregate originator messages
- originator interval -> Orginater interval
- bonding mode -> Bonding mode

Reply:
OK